signer/local/local.go

```diff
@@ -254,7 +257,53 @@ func (s *Signer) Sign(req signer.SignRequest) (cert []byte, err error) {
 		safeTemplate.SerialNumber = serialNumber
 	}
 
-	return s.sign(&safeTemplate, profile)
+	var certTBS = safeTemplate
+	if len(profile.CTLogServers) == 0 {
+		cert, err = s.sign(&certTBS, profile)
+		return
+	}
+
```

Comment:
the below block of dealing with CTLogServers can be included with "if len(profile.CTLogServers) > 0", so we don't need to write the bail-out early branch logic above.